### PR TITLE
[SEMI-MODULAR][HELP NEEDED] Species Module, Tajaran and Vulpkanin

### PR DESCRIPTION
**Pull request by LYNXEMS**
## About The Pull Request

Adds the species module, Tajaran and Vulpkanin

## Why It's Good For The Game

Species on the roadmap and diversity

## Changelog

:cl:
add: Tajaran and Vulpkanin
/:cl:

### CODE REVIEW

File: code/__DEFINES/~pariah_defines/DNA.dm [hunk 1]
```diff
@@ -17,3 +17,9 @@
 
 #define MANDATORY_FEATURE_LIST list("flavor_text" = "")
 
+//Species
+#define DNA_TAJARAN_MARKINGS_BLOCK 17
+#define DNA_TAJARAN_TAIL_BLOCK 18
+#define DNA_TAJARAN_SNOUT_BLOCK 19
```
Comment (Kapu1178) [inline code/__DEFINES/~pariah_defines/DNA.dm:23]: dont modularize these, these need to be where everything else is so people dont add dna blocks with the same values

File: code/__DEFINES/~pariah_defines/DNA.dm [hunk 2]
```diff
@@ -17,3 +17,9 @@
...
+
+#define ORGAN_SLOT_EXTERNAL
```
Comment (Kapu1178) [inline code/__DEFINES/~pariah_defines/DNA.dm:25]: im assuming this is an accident

File: code/modules/mob/living/carbon/human/species.dm [hunk 3]
```diff
@@ -41,6 +41,14 @@ GLOBAL_LIST_EMPTY(features_by_species)
...
 	///Never, Optional, or Forced digi legs?
 	var/digitigrade_customization = DIGITIGRADE_NEVER
+
+	// PARIAH EDIT BEGIN - DIGITGRADE
+	///Path to Right Digi leg
+	var/digitigrade_r_leg = /obj/item/bodypart/r_leg/digitigrade
+	///Path to Left Digi leg
+	var/digitigrade_l_leg = /obj/item/bodypart/l_leg/digitigrade
+	// PARIAH EDIT END - DIGITIGRADE
+
```
Comment (Kapu1178) [inline code/modules/mob/living/carbon/human/species.dm:51]: this should be a var on `/datum/bodypart/[side]/digitigrade`

File: config/game_options.txt [hunk 4]
```diff
@@ -314,12 +314,13 @@ SILICON_MAX_LAW_AMOUNT 12
 ROUNDSTART_RACES human
 
 ## Races that are strictly worse than humans that could probably be turned on without balance concerns
-ROUNDSTART_RACES felinid
+#ROUNDSTART_RACES felinid
```
Comment (Kapu1178) [inline config/game_options.txt:317]: Undo this

File: modular_pariah/modules/species/code/modules/surgery/bodyparts/species_parts/tajaran_bodyparts.dm [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+		if(!(human_owner.w_uniform) || (human_owner.w_uniform.supports_variations_flags & (CLOTHING_DIGITIGRADE_VARIATION|CLOTHING_DIGITIGRADE_VARIATION_NO_NEW_ICON))) //Checks uniform compatibility
+			uniform_compatible = TRUE
+		if((!human_owner.wear_suit) || (human_owner.wear_suit.supports_variations_flags & (CLOTHING_DIGITIGRADE_VARIATION|CLOTHING_DIGITIGRADE_VARIATION_NO_NEW_ICON)) || !(human_owner.wear_suit.body_parts_covered & LEGS)) //Checks suit compatability
+			suit_compatible = TRUE
+
+		if((uniform_compatible && suit_compatible) || (suit_compatible && human_owner.wear_suit?.flags_inv & HIDEJUMPSUIT)) //If the uniform is hidden, it doesnt matter if its compatible
+			limb_id = "digitigrade_tajaran"
+
+		else
+			limb_id = "tajaran"
```
Comment (Kapu1178) [inline modular_pariah/modules/species/code/modules/surgery/bodyparts/species_parts/tajaran_bodyparts.dm:78]: just make these a subtype of the existing digitigrade legs.

**Review by Kapu1178 — CHANGES_REQUESTED** [review]

File: config/game_options.txt (hunk 4 above)
Comment (ElorgRHG) [reply config/game_options.txt:317]: :trollface:

**Comment by LYNXEMS** [comment]
Lil unsure how to fix the linter thing, the digitigrade is to give the legs a potential variant for each species, since right now it is a little hardcoded for lizards.

**Pull request closed without merging**